Sources/FaunaDB/Value.swift

```diff
@@ -331,45 +369,58 @@ extension BytesV: Equatable {
     }
 }
 
+/// Represents a query value in the FaunaDB query language.
+/// [Reference](https://fauna.com/documentation/queries#values-special_types).
+public struct QueryV: ScalarValue, AsJson {
+
+    public let value: [String: Any]
```

Comment:
The problem of representing as `[String: Value]` is that the body of lambda would be wrapped as `ObjectV` after parsed, so when we send it back to the server it would be wrapped with `{"object": {}}`